pyproject.toml

```diff
@@ -60,7 +60,7 @@ platforms = ["osx-arm64", "linux-64"]
 dev = { features = ["dev"], solve-group = "default" }
 
 [tool.pixi.tasks]
-example = "pdal_pipeline create-dsm  notebooks/uw-campus.geojson notebooks/UTM_10N_WGS84_G2139_3D.wkt /tmp/dem"
+example = "pdal_pipeline create-dsm notebooks/uw-campus.geojson /tmp/dem notebooks/UTM_10N_WGS84_G2139_3D.wkt"
```

Comment:
Note: to avoid future ordering issues and also for better transparency we can also use explicit argument flags for these example commands, for example:

> pdal_pipeline create-dsm --no-cleanup --local-laz-dir /tmp/lidar_tools/WA_KingCounty_2021_B21 --extent-polygon notebooks/uw-campus.geojson --target-wkt notebooks/UTM_10N_WGS84_G2139_3D.wkt --output-prefix /tmp/test